monkeys/docker/config/config.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Should this file be here? Doesn't it contain sensitive data?

Reply:
> Should this file be here? Doesn't it contain sensitive data?

You're right! :|